historic_data_import/management/commands/data_import.py

```diff
@@ -0,0 +1,236 @@
+from typing import Optional
+
+from django.core.management.base import BaseCommand
+from django.db import IntegrityError, transaction
+
+from shedpi_hub_dashboard.models import Device, DeviceModule, DeviceModuleReading
+
+# TODO: Logger
+
+"""
+A script ot import the data that has already been gathered
+
+Historic data structure:
+    "Shed pi started: {get_time()}, using version: 0.0.1"
+    "Pi temp: {pi_temp}, probe_1 temp: {probe_1_temp}"
+
+FIXME:
```

Comment:
Potentially still to do, worth breaking the data import to come after splitting the modules